### PR TITLE
feat: align the variables.css with colors.css in ui-components

### PR DESCRIPTION
**Pull request by Sboonny**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

While working on the add system opt in PR #48757, I found variables that aren't used. If you find something broken, then they are used, I don't know where tho 😅

### CODE REVIEW

**Comment by moT01** [comment]
I feel like I would just leave these in there, they are part of our [design style guide](https://design-style-guide.freecodecamp.org/) - so if someone is ever looking for a color to use for something, these options will already be available in the palettes.

**Comment by moT01** [comment]
I would just leave it for now and see what @ahmadabdolsaheb thinks. We probably could add those reds to the design style guide and the color palette (variables.css)

Removing these here really doesn't have any consequences so I don't mind - I just prefer to leave them.

**Comment by ahmadabdolsaheb** [comment]
Some of these colors are such as '--green-light: #acd157;' are used in the client and will be used in component library.
Others are coming from the style guide and have been added to future proof the color variables.
I would wait until all the components from learn have moved to the component library to remove the unused variables.

File: client/src/components/solution-display-widget/index.tsx [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { Button, Dropdown, MenuItem } from '@freecodecamp/react-bootstrap';
+import { Button } from '@freecodecamp/react-bootstrap';
+import { Dropdown, MenuItem } from '@freecodecamp/ui';
 import React from 'react';
 import { useTranslation } from 'react-i18next';
 import { CompletedChallenge } from '../../redux/prop-types';
 import { getSolutionDisplayType } from '../../utils/solution-display-type';
 import './solution-display-widget.css';
+import '@freecodecamp/ui/dist/bundle';
```
Comment (Sboonny) [inline client/src/components/solution-display-widget/index.tsx:10]: shouldn't be merged in

File: client/package.json [hunk 2]
```diff
@@ -49,6 +49,7 @@
     "@freecodecamp/react-bootstrap": "0.32.3",
     "@freecodecamp/react-calendar-heatmap": "1.1.0",
     "@freecodecamp/strip-comments": "3.0.1",
+    "@freecodecamp/ui": "^0.0.1",
```
Comment (Sboonny) [inline client/package.json:52]: shouldn't be merged in

**Comment by Sboonny** [comment]
To test this
- revert [ea449a3](https://github.com/freeCodeCamp/freeCodeCamp/pull/49273/commits/ea449a3954f88937754caf01be5696a85cb9bd23)
- build the ui-component in `./tools/ui-component`
- import a component that need night and light themes from the ui-component, most of the buttons components
- check the page in light and dark theme

**Pull request closed without merging**